nucypher/blockchain/eth/constants.py

```diff
@@ -54,4 +54,4 @@
     80002,  # "Polygon/Amoy"
 }
 
-CHAINLIST_URL = "https://raw.githubusercontent.com/nucypher/chainlist/main/rpc.json"
+CHAINLIST_URL = "https://raw.githubusercontent.com/nucypher/chainlist/main/"
```

Comment:
What about making this string templated, assuming the {domain}.json is the only use of it? It may help to prevent misuse.

```suggestion
CHAINLIST_URL = "https://raw.githubusercontent.com/nucypher/chainlist/main/{domain}.json"
```